client-app/src/desktop/AppModel.ts

```diff
@@ -47,6 +59,7 @@ export class AppModel extends BaseAppModel {
     override async initAsync() {
         await super.initAsync();
         await XH.installServicesAsync(GitHubService, PortfolioService);
+        XH.router.setOption('queryParamsMode', 'loose');
```

Comment:
What's this do? Could you add a comment? Easy to miss if it's important to the functioning of the example (which I assume it is)

Reply:
Per John's comment above: 

``` 
This also requires putting the router query params in "loose" mode (which just means you can put whatever you want in the params and router5 wont validate/modify/etc.)

XH.router.setOption('queryParamsMode', 'loose');
```

Will add comment above this line to explain in the code.